src/config_default_credentials.rs

```diff
@@ -14,7 +15,10 @@ use crate::{Error, TokenProvider};
 
 /// A token provider that uses the default user credentials
 ///
-/// Reads credentials from `.config/gcloud/application_default_credentials.json`.
+/// Reads credentials from `.config/gcloud/application_default_credentials.json` on Linux and MacOS
+/// or from `%APPDATA%/gcloud/application_default_credentials.json` on Windows.
+/// See [GCloud Application Default Credentials](https://cloud.google.com/docs/authentication/application-default-credentials#personal)
+/// for details
```

Comment:
Nit: please end this sentence in a period.

Reply:
Done.